src/state/user/reducer.ts

```diff
@@ -75,6 +87,16 @@ export default createReducer(initialState, builder =>
             delete state.tokens[ChainId.MAINNET][WETH[ChainId.MAINNET].address]
           }
         }
+
+        // slippage isnt being tracked in local storage, reset to default
+        if (!!!state.userSlippageTolerance) {
```

Comment:
what if the user sets to 0? we should probably do `if (typeof state.userSlippageTolerance !== 'number')`